### ISSUE TITLE
Parse error: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\xampp\htdocs\vendor\tmarois\filebase\src\Database.php on line 88

### ISSUE THREAD

**Issue by Jtreix** [tmarois/Filebase#15]
Parse error: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\xampp\htdocs\vendor\tmarois\filebase\src\Database.php on line 88
how can i fixed this ?

**Comment by timothymarois** [comment]
Do you have the format class set in your database config.

```
$db = new \Filebase\Database([
    'format'  => \Filebase\Format\Json::class
]);
```

**Comment by timothymarois** [comment]
I have pushed a new update which should fix the error you received, version 1.0.16. This was originally designed primarily using php 7, I have tested and confirmed things are working on my end with php 5.6.32, let me know if you find any more issues.
